### DISCUSSION TITLE
Error while transferring files - Log?

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [iterate-ch/cyberduck#16501] [Q&A]
Hi,

I transferred files into a Cryptomator Vault via WebDav.
In the transfer windows window it shows that that not everything was copied but I see nowhere a possibility to actually see what failed.
What am I missing?

Thanks
Hyper

**Comment by Only-Bimal** [comment]
Facing the same issue while syncing with onedrive. All the transfers end as "Transfer Incomplete" but there is nothing in the log pane. also didn't found any way to know why the transfer is incomplete

**Reply by sergiks** [reply]
Same issue. Downloaded multiple folders from Amazon S3, some files have failed. Reasons:
1. restoration from Glacier Deep Archive was not yet completed for some items;
2. faulty prefix, containing 2 consecutive slashes like `/folder_1/folder_2//filename.ext`

Picked "Continue", "all" option in the UI. 

Transfer took several hours - a lot of large files.

Now I want to see which exactly files have failed and were ignored by the "Continue" option. 
How can I do that?
